cms/templates/cms/course_team_tab_page.html

```diff
@@ -8,136 +8,137 @@
+                <span class="name">{{ instructor.value.name }}</span>
+                {{ instructor.value.title }}
               </div>
             </div>
-            <div class="profile-content">
-              <span class="name">{{ instructor.value.name }}</span>
-              {{ instructor.value.title }}
+          </li>
+          {% endfor %}
+        </ul>
+      </section>
+    {% endif %}
+
+    {% if child_page.administrators|length > 0 %}
+      <section id="administrators" class="team-section">
+        <h1 class="section-title">{{ child_page.administrators_heading }}</h1>
+        <ul class="team-list-administrator">
+          {% for administrator in child_page.administrators %}
+          <li class="team-list-item">
+            <div class="admin-team-list-item">
+                <div class="profile-image-wrapper">
+                  <img class="member-image" src="{{ administrator.value.image.file.url }}" alt="">
+                  <div class="bio-content">
+                    <p class="bio">{{ administrator.value.bio }}</p>
+                  </div>
+                </div>
+                <div class="profile-content">
+                  <span class="name">{{ administrator.value.name }}</span>
+                  {{ administrator.value.title }}
+                </div>
             </div>
-          </div>
-        </li>
-        {% endfor %}
-      </ul>
-    </section>
-  {% endif %}
-
-  {% if child_page.administrators|length > 0 %}
-    <section id="administrators" class="team-section">
-      <h1 class="section-title">{{ child_page.administrators_heading }}</h1>
-      <ul class="team-list-administrator">
-        {% for administrator in child_page.administrators %}
-        <li class="team-list-item">
-          <div class="admin-team-list-item">
+          </li>
+          {% endfor %}
+        </ul>
+      </section>
+    {% endif %}
+
+    {% if child_page.contributors|length > 0 %}
+      <section id="contributors" class="team-section">
+        <h1 class="section-title">{{ child_page.contributors_heading }}</h1>
+        <ul class="team-list-administrator">
+          {% for contributor in child_page.contributors %}
+          <li class="team-list-item">
+            <div class="admin-team-list-item">
               <div class="profile-image-wrapper">
-                <img class="member-image" src="{{ administrator.value.image.file.url }}" alt="">
+                <img class="member-image" src="{{ contributor.value.image.file.url }}" alt="" />
                 <div class="bio-content">
-                  <p class="bio">{{ administrator.value.bio }}</p>
+                  <p class="bio">{{ contributor.value.bio }}</p>
                 </div>
               </div>
               <div class="profile-content">
-                <span class="name">{{ administrator.value.name }}</span>
-                {{ administrator.value.title }}
+                <span class="name">{{ contributor.value.name }}</span>
```

Comment:
Why this change and where is it reflected in the back end?

Reply:
I think this is something because of moving `divs' inside a container, I can see any reflections of this change on the page.